.gitignore

```diff
@@ -1 +1,2 @@
 exampleSite/themes/hugo-theme-lean-launch-page
+.DS_Store
```

Comment:
Can you fix this please? Either remove the line completely (this is a mac thing) or at least fix the line ending so this error goes away.

Reply:
The error doesn't show in this view but you if view the "Files Changed" tab you'll see it.

Reply:
Looks good now.